docs/_documentations/mdt-vsc-nodejsprofiling.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: docs
+title: Installing and running the Codewind language server for Node.js profiling
+description: Installing and running the Codewind language server for Node.js profiling
+keywords: Node.js, Codewind, server, language, languages, code, code highlighting, highlighting, profiling, Visual Studio Code, VS Code, JavaScript
+duration: 1 minute
+permalink: mdt-vsc-nodejsprofiling
+type: document
+---
+
+# Installing and running the Codewind language server for Node.js profiling
+The Codewind language server for Node.js profiling annotates your Node.js code with code highlighting. Use profiling data gathered through Codewind load testing, and the highlighting shows the relative time that is spent in JavaScript functions.
+
```

Comment:
doesn't read quite right,   'Use profiling data gathered through Codewind load testing, and the highlighting shows the relative time that is spent in JavaScript functions.'

Reply:
@tobespc Would it be best to simply remove this sentence, or do you have suggestions on how we could make it more accurate? Thanks!

Reply:
@tobespc @sishida How about:
The Codewind language server for Node.js profiling annotates your Node.js code with code highlighting. Code highlighting uses the profiling data gathered through Codewind load testing to highlight and show the relative time that is spent in JavaScript functions.

Reply:
thats fine

Reply:
@micgibso and @tobespc I have made the changes in the latest commit.